src/pages/Game.js

```diff
@@ -16,9 +20,12 @@ export const Game = () => {
       move: { piece: X, position: 5 },
     })
   }
+  const { gameId } = useParams()
 
   const callback = useCallback((gameState) => {
     console.log('GAME_STATE', gameState)
+    setBoard(gameState.board)
+    setInfo({ players: gameState.players, playerTurn: gameState.playerTurn })
```

Comment:
Again, this works, but lets try to use destructuring where we can in order to make our code more readable:
```js
  const callback = useCallback(({ board, players, playerTurn }) => {
    setBoard(board)
    setInfo({ players, playerTurn })
  }, [])
```